packages/server/src/server.ts

```diff
@@ -104,6 +175,8 @@ rpc.use('eth_getCode', async (params: any) => {
  * returns: Chain ID - integer
  */
 rpc.use('eth_chainId', async () => {
+  methodCounter.inc();
```

Comment:
We don't want this. Front ends can aggregate across tags.

Reply:
Good point. Removed